tutorials/applications/recipes/loading_text_files.ipynb

```diff
@@ -0,0 +1,401 @@
+{
```

Comment:
<p>I think you can condense everything by taking in the ExecutionEngine and then you can have if-else inside <code>read_text_file</code> to change behavior depending on the engine given.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fugue-project/tutorials/pull/178/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>